src/main/java/edu/wpi/first/pathweaver/Game.java

```diff
@@ -86,15 +87,15 @@ public int hashCode() {
     return games;
   }
 
-  private static Game loadPowerupGame() {
+  private static Game loadGameFromResource(String gameJson) {
```

Comment:
Maybe `gameJsonPath`? `gameJson` implies that it expects a JSON-formatted string instead of a path to the json file